config/forms/2021_2022.json

```diff
@@ -2022,40 +2232,403 @@
                 },
                 "tcharge": {
                   "check_answer_label": "Total Charge",
-                  "header": "Total charge?",
-                  "hint_text": "This is the total of rent and all charges",
+                  "header": "Total charge",
+                  "hint_text": "This is the total for rent and all charges.",
                   "type": "numeric",
                   "min": 0,
                   "step": 1,
                   "prefix": "£",
                   "suffix": "every week",
                   "readonly": true,
                   "requires_js": true
+                }
+              },
+              "depends_on" : [{
```

Comment:
Might need to add more logic to evaluating depends_on instead of writing out every condition

Reply:
Think this is fine. It's easily understandable.